config/sources/families/include/meson64_common.inc

```diff
@@ -168,11 +168,42 @@ family_tweaks_bsp()
 	install -m 755 $SRC/packages/bsp/rockchip/hdmi-hotplug $destination/usr/local/bin
 
 	mkdir -p "$destination"/etc/X11/xorg.conf.d
+	if [[ $BOARD == odroidc2 || $BOARD == nanopik2 || $BOARD == lepatato || $BOARD == lafrite || $BOARD == vim1 ]]; then 
 	cat <<-EOF > "$destination"/etc/X11/xorg.conf.d/02-driver.conf
 	# set fbdev as default driver.
 	Section "Device"
-         Identifier "NOGPU"
-         Driver "fbdev"
-	EndSection
+			Identifier "NOGPU"
+			Driver "fbdev"
+		EndSection
 	EOF
+	else
+	if [[ $BOARD == odroidn2 || $BOARD == odroidc4 || $BOARD == vim2 ]]; then
```

Comment:
EOF
	else
	if

	EOF
	elif [[

Reply:
it was failing when I used elif.  if kept saying it could not find the fi .